tests/util/defs.bzl

```diff
@@ -24,10 +25,15 @@ def _directory_impl(ctx):
     args = ctx.actions.args()
     args.add(out_dir_file.path)
 
+    # This helper is horrible.  We should pass all the args in files.
```

Comment:
🙂

Reply:
It's more of a reminder to future maintainer that they may have to do something more robust one day.